.github/workflows/integration.yml

```diff
@@ -262,8 +262,10 @@ jobs:
           composer -d src/Elasticsearch config version ${SW_RECOVERY_NEXT_VERSION}
 
       - name: Build updater
-        working-directory: new-shopware/src/WebInstaller
+        working-directory: new-shopware/src/
         run: |
+          git clone https://github.com/shopware/web-installer.git WebInstaller
```

Comment:
maybe also test the latest released version?

Reply:
clone the latest tag?

Reply:
I just grab the latest phar, why build it lol